cypress/integration/userCanSelectActivities.feature.js

```diff
@@ -0,0 +1,17 @@
+describe("Activities", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/activities");
```

Comment:
can user pick activities for a trip without a destination?

Reply:
No. adjusts tests to reflect that.